tests/performance/specs/single-post.test.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * WordPress dependencies
+ */
+import { test } from '@wordpress/e2e-test-utils-playwright';
+
+/**
+ * Internal dependencies
+ */
+import { camelCaseDashes } from '../utils';
+
+const results = {
+	timeToFirstByte: [],
+	largestContentfulPaint: [],
+	lcpMinusTtfb: [],
+};
+
+const themes = [ 'twentytwentyone', 'twentytwentythree', 'twentytwentyfour', 'twentytwentyfive' ];
+
+const locales = [ 'en_US', 'de_DE' ];
+
+test.describe( 'Single Post', () => {
+	test.use( {
+		storageState: {}, // User will be logged out.
+	} );
+
+	for ( const theme of themes ) {
+		for ( const locale of locales ) {
+			test.describe( `Theme: ${ theme }, Locale: ${ locale }`, () => {
+				test.beforeAll( async ( { requestUtils } ) => {
+					await requestUtils.activateTheme( theme );
+					await requestUtils.updateSiteSettings( {
+						language: 'en_US' === locale ? '' : locale,
+					} );
+				} );
+
+				test.afterAll( async ( { requestUtils }, testInfo ) => {
+					await testInfo.attach( 'results', {
+						body: JSON.stringify( results, null, 2 ),
+						contentType: 'application/json',
+					} );
+
+					await requestUtils.updateSiteSettings( {
+						language: '',
+					} );
+
+					results.largestContentfulPaint = [];
+					results.timeToFirstByte = [];
+					results.lcpMinusTtfb = [];
+				} );
+
+				const iterations = Number( process.env.TEST_RUNS );
+				for ( let i = 1; i <= iterations; i++ ) {
+					test( `Measure load time metrics (${ i } of ${ iterations })`, async ( {
+						page,
+						metrics,
+					} ) => {
+						await page.goto( '/?clear_cache' );
+						await page.goto( '/2018/11/03/block-image/' );
```

Comment:
Why do you hit the home page with the `clear_cache` parameter first? Shouldn't the actual post URL instead have the `clear_cache` parameter?

Reply:
This is not just a query parameter for cache busting. This triggers the `clear-cache.php` mu-plugin to reset things like object cache and then `exit()`. It's supposed to be used separately.

Reply:
Ah, I see, had missed the `exit` statement. Maybe add a comment over each of these calls to clarify this is not actually to "visit" the page?

E.g.:
```js
// This call simply resets caches and does not actually load the home page.
```